docs/content/en/docs/getting-started/metrics/_index.md

```diff
@@ -4,3 +4,321 @@ description: Learn how Keptn metrics enhances your deployment
 weight: 25
 ---
 
+The Keptn metrics component of the Keptn Lifecycle Toolkit
+allow you to define any type of metric
+from multiple instances of any type of data source in your Kubernetes cluster.
+You may have tools like Argo, Flux, KEDA, HBA, Keptn
+that need observability data to make automated decisions.
+Whether a rollout is good, whether to scale up or down.
+Your observability data may come
+from multiple observability solutions --
+Datadog, Dynatrace, data in AWS, Google, and Azure --
+and include data in Lifestep (?) and Honeycomb or Splunk.
+
+## Using this exercise
+
+This exercise is based on the
+[simplenode-dev](https://github.com/keptn-sandbox/klt-on-k3s-with-argocd)
+example.
+You can clone that repo to access it locally
+or just look at it for examples
+as you implement the functionality "from scratch"
+on your local Kubernetes deployment cluster.
+
+The steps to implement pre- and post-deployment orchestration are:
+
+1. [Bring or create a Kubernetes cluster](#bring-or-create-a-kubernetes-deployment-cluster)
+1. [Install the Keptn Lifecycle Toolkit on your cluster](#install-klt-on-your-cluster)
+1. [Enable KLT for your cluster](#enable-klt-for-your-cluster)
+1. [Integrate KLT with your cluster](#integrate-klt-with-your-cluster)
+1. Define metrics to use
+   * [Define metrics providers](#define-metrics-providers)
+   * [Define KeptnMetric information](#define-keptnmetric-information)
+   * [View available metrics](#view-available-metrics)
+
+See the
+[Introducing Keptn Lifecycle Toolkit](https://youtu.be/449HAFYkUlY)
+video for a demonstration of this exercise.
+
+## Bring or create a Kubernetes deployment cluster
+
+You can run this exercise on an existing Kubernetes cluster
+or you can create a new cluster.
+For personal study and demonstrations,
+this exercise runs well on a local Kubernetes cluster.
+See [Bring or Install a Kubernetes Cluster](../../install/k8s.md).
+
+## Install KLT on your cluster
+
+Install the Keptn Lifecycle Toolkit on your cluster
+by executing the following command sequence:
+
+```shell
+helm repo add klt https://charts.lifecycle.keptn.sh
+helm repo update
+helm upgrade --install keptn klt/klt \
+   -n keptn-lifecycle-toolkit-system --create-namespace --wait
+```
+
+If you only want to use Keptn's metrics features,
+you can install just the `metrics-operator`
+bu modifying Helm values.
+See
+[Install KLT](../../install/install.md)
+for more information about installing the Lifecycle Toolkit.
+
+To verify that the `metrics-operator` is installed in your cluster,
+run the following command:
+
+```shell
+kubectl get pods -n keptn-lifecycle-toolkit-system
+```
+
+The output shows all components that are running on your system.
```

Comment:
I would create a snippet and cross-reference it from here and the install page. This way, we don't need to maintain the instruction on how to install KLT in multiple places. You can see an example how to include snippets here: https://github.com/thisthat/lifecycle-controller/blob/c5c45558795d99a7789c48b9940d336990f46d3e/docs/content/en/docs/getting-started/generic-gs/_index.md?plain=1#L43